pkg/payload/precondition/clusterversion/upgradeable.go

```diff
@@ -66,9 +82,21 @@ func (pf *Upgradeable) Run(ctx context.Context, releaseContext precondition.Rele
 	desiredMinor := getEffectiveMinor(releaseContext.DesiredVersion)
 
 	// if there is no difference in the minor version (4.y.z where 4.y is the same for current and desired), then we can still upgrade
+	// if no cluster overrides have been set
 	if currentMinor == desiredMinor {
 		klog.V(4).Infof("Precondition %q passed: minor from the current %s matches minor from the target %s (both %s).", pf.Name(), cv.Status.History[0].Version, releaseContext.DesiredVersion, currentMinor)
-		return nil
+		if condition := ClusterVersionOverridesCondition(clusterVersion); condition != nil {
+			klog.V(4).Infof("Update from %s to %s blocked by %s: %s", cv.Status.History[0].Version, releaseContext.DesiredVersion, condition.Reason, condition.Message)
+
+			return &precondition.Error{
+				Nested:  err,
```

Comment:
we should drop this line right?  Because `err` was last-touched way up at `pf.lister.Get`, and we already handled the `err != nil` case there.  This condition error is born here, and not wrapping an underlying error.

Reply:
removed in [9336d69](https://github.com/openshift/cluster-version-operator/pull/364/commits/9336d69bb60bf43d4a5a60e403a3c28a7016c2f9)